indy_common/types.py

```diff
@@ -303,6 +307,58 @@ class ClientPoolConfigOperation(MessageValidator):
     )
 
 
+class ConstraintField(MessageValidator):
```

Comment:
Please add tests for `ConstraintField`, `ConstraintEntityField` and `ConstraintListField`